cfme/tests/infrastructure/test_vm_ownership.py

```diff
@@ -85,3 +91,35 @@ def test_set_vm_owner(self, appliance, vm, from_detail):
         rest_vm.reload()
         assert hasattr(rest_vm, "evm_owner_id")
         assert rest_vm.evm_owner.userid == "admin"
+
+
+@pytest.fixture(scope='module')
+def small_vm(provider, small_template_modscope):
+    vm = provider.appliance.collections.infra_vms.instantiate(random_vm_name(context='rename'),
+                                                              provider,
+                                                              small_template_modscope.name)
+    vm.create_on_provider(find_in_cfme=True, allow_skip="default")
+    vm.refresh_relationships()
+    yield vm
+    vm.cleanup_on_provider()
+
+
+@pytest.mark.ignore_stream('5.9')
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE_PER_TYPE)
+def test_rename_vm(appliance, setup_provider, small_vm):
+
+    """Test for rename the VM.
+       This feature is included in 5.10z.
+    Steps:
+    1. Add VMware provider
+    2. Provision VM
+    3. Navigate to details page of VM
+    4. Click on Configuration > Rename this VM > Enter new name
+    5. Click on submit
+    6. Check whether VM is renamed or not
+    """
+    view = navigate_to(small_vm, 'Details')
+    small_vm.rename_vm(new_vm_name="test-{}".format(fauxfactory.gen_alphanumeric()))
+    view.wait_displayed("60s")
+    msg = 'Rename of Virtual Machine "{vm_name}" has been initiated'.format(vm_name=small_vm.name)
+    assert view.flash.read()[0] == msg
```

Comment:
Question:
Why you are asserting in this way?

Reply:
@digitronik 
Yes, I changed the way of asserting and used **view.flash.assert_success_message()**.